monitoring/values-prom-operator.yaml

```diff
@@ -237,3 +237,11 @@ grafana:
       - kubernetes.io/pvc-protection
     # subPath: ""
     # existingClaim:
+
+tempo: 
```

Comment:
I don't see a tempo: stanza available in the [kube-prometheus-stack values.yaml](https://github.com/grafana/helm-charts/blob/main/charts/grafana/values.yaml) file nor in the [grafana values.yaml](https://github.com/grafana/helm-charts/blob/main/charts/grafana/values.yaml) file.  Should this be in the values-tempo.yaml file?